pkgs/games/xu4/default.nix

```diff
@@ -0,0 +1,88 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, makeWrapper
+, makeDesktopItem
+, zlib
+, libpng
+, boron
+, allegro5
+, libGL
+, libX11
+, libXxf86vm
+, libXcursor
+, libxml2
+, faun
+}:
+
+stdenv.mkDerivation rec {
+  pname = "xu4";
+  version = "1.3";
+
+  src = fetchFromGitHub {
+    owner = "xu4-engine";
+    repo = "u4";
+    rev = "v${version}";
+    hash = "sha256-3jyUSVfmjuKsHQKluFnQ8Gtg7aAI/WBjmY109BdNdyY=";
+    fetchSubmodules = true;
+  };
+
+  # this is not a standard Autotools-like `configure` script
+  dontAddPrefix = true;
+
+  preConfigure = ''
+    patchShebangs configure
+  '';
+
+  postConfigure = ''
+    # need to report this upstream
+    sed --in-place 's/-Wall/-Wno-error=format-security/g' src/Makefile
+
+    # disable git submodule init as part of build
+    sed --in-place 's/^\s*git submodule.*//g' src/Makefile.common
+  '';
+
+  enableParallelBuilding = true;
+
+  buildInputs = [
```

Comment:
The allegro5 and libxml2 inputs are not needed here (or in the list above).

Reply:
Removed in https://github.com/NixOS/nixpkgs/pull/181866/commits/571e02812f5da12fd851557ebbacea72fc7b5121